js/ai/src/formats/types.d.ts

```diff
@@ -14,7 +14,8 @@
  * limitations under the License.
  */
 
-import { GenerateResponse, GenerateResponseChunk } from '../generate';
+import { GenerateResponse } from '../generate';
+import { GenerateResponseChunk } from '../generate/chunk';
```

Comment:
Should this line have changed? I see both `GenerateResponse` and `GenerateResponseChunk` are still exported from `generate`.

Reply:
Good catch.